cli/vm/vm.go

```diff
@@ -27,10 +28,13 @@ func startVMPrompt(ctx *cli.Context) error {
 	if err := cmdargs.EnsureNone(ctx); err != nil {
 		return err
 	}
-
-	cfg, err := options.GetConfigFromContext(ctx)
-	if err != nil {
-		return cli.NewExitError(err, 1)
+	var err error
+	cfg := config.VM
```

Comment:
Try to unify the style of declarations that are grouped. It's either `var` or `:=`.